lib/Runtime/PlatformAgnostic/Platform/CMakeLists.txt

```diff
@@ -24,6 +24,12 @@ set(PL_SOURCE_FILES ${PL_SOURCE_FILES}
   )
 endif()
 
+if((CC_TARGET_OS_LINUX OR CC_TARGET_OS_OSX) AND IntlICU)
+set(PL_SOURCE_FILES ${PL_SOURCE_FILES}
+  Common/Intl.cpp
+  )
+endif()
```

Comment:
Not that we use CMake on Windows, but is there any other reason for OS-guarding this?

Reply:
Good point.

Reply:
Done